tracee-rules/benchmark/benchmark_test.go

```diff
@@ -175,9 +177,10 @@ func BenchmarkEngineWithNSignatures(b *testing.B) {
 	for _, bc := range benches {
 		for _, tc := range bc.sigCount {
 			b.Run(fmt.Sprintf("%s/%dSignatures", bc.name, tc), func(b *testing.B) {
-				sig, _ := bc.sigFunc()
```

Comment:
We always ended up with 1 channel for N signatures due to this bug. It was pretty hard to catch so I also added an assert to the `engine.NewEngine` constructor.